debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -173,14 +174,15 @@ public static PostgresValueConverter of(PostgresConnectorConfig connectorConfig,
                 connectorConfig.binaryHandlingMode(),
                 connectorConfig.intervalHandlingMode(),
                 connectorConfig.getUnavailableValuePlaceholder(),
+                connectorConfig.getUnavailableNumberArrayPlaceholder(),
                 connectorConfig.moneyFractionDigits());
     }
 
     protected PostgresValueConverter(Charset databaseCharset, DecimalMode decimalMode,
                                      TemporalPrecisionMode temporalPrecisionMode, ZoneOffset defaultOffset,
                                      BigIntUnsignedMode bigIntUnsignedMode, boolean includeUnknownDatatypes, TypeRegistry typeRegistry,
                                      HStoreHandlingMode hStoreMode, BinaryHandlingMode binaryMode, IntervalHandlingMode intervalMode,
-                                     byte[] toastPlaceholder, int moneyFractionDigits) {
+                                     byte[] toastPlaceholder, String toastPlaceholderNumberArrayString, int moneyFractionDigits) {
```

Comment:
If there will be additional placeholders it is not good to pass them as extended list of params.
In that case we should introduce a class that will manage all placeholder types and puss just that one.